languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/style.xml

```diff
@@ -23265,7 +23266,7 @@ USA
                 <example correction="afin de|pour|de façon à|ainsi">Il le dit <marker>de manière à</marker> réussir son projet.</example>
             </rule>
         </rulegroup>
-        <rulegroup id="REP_CONCEQUENCE" name="conséquence (style)" min_prev_matches="1">
+        <rulegroup id="REP_CONSEQUENCE" name="conséquence (style)" min_prev_matches="1">
```

Comment:
Nitpick: maybe that shouldn't be changed (or, if we change the name, we should also maybe change the rule itself). The rule is often disabled, and changing the name will make it pop up again for the users...
![image](https://github.com/languagetool-org/languagetool/assets/114988314/5e43dde3-4e7a-49e2-93ff-643c1394a6c6)

Reply:
Ok, so I've reverted the typo correction and the rule is now picky, we will focus on picky, we will see what to do with this kind of repetition rules wrt picky tag later this quarter I think